qiskit/pulse/macros.py

```diff
@@ -63,16 +63,11 @@ def measure(
 
     # backend is V2.
     if hasattr(backend, "target"):
-        try:
-            meas_map = backend.configuration().meas_map
-        except AttributeError:
-            # TODO add meas_map to Target in 0.25
-            meas_map = [list(range(backend.num_qubits))]
 
         return _measure_v2(
             qubits=qubits,
             target=backend.target,
-            meas_map=meas_map,
+            meas_map=meas_map or getattr(backend, "meas_map", [list(range(backend.num_qubits))]),
```

Comment:
This attribute is defined in the base class of V2 backend, so it's fair to assume backend has meas_map. On the other hand, it is not implemented and raises `NotImplementedError` by default, so 
```python
try:
    meas_map = backend.meas_map
except NotImplementedError:
    meas_map = [list(range(backend.num_qubits))]
```
is more appropriate. Do you have any reason to use `getattr`?
https://github.com/Qiskit/qiskit-terra/blob/02502b5d98796dcc2c8b80ac5d0f2af85e978e5c/qiskit/providers/backend.py#L452-L466

Reply:
Fixed at https://github.com/Qiskit/qiskit-terra/pull/10135/commits/4ae58e722fb9f5eee34b2ee7cec897d8e48b51bf.